### DISCUSSION TITLE
Netbox Update

### DISCUSSION THREAD

**Discussion by mm3906078** [netbox-community/netbox#13128] [Help Wanted!]
Hi guys
I want to update my Netbox to the latest (now it's 3.5.5) but upgrade.sh stuck on 3.5.0, Does anyone have any idea?
I have my Netbox Ubuntu 20.04.6 LTS and I have Python 3.8

**Comment by candlerb** [comment]
"upgrade.sh stuck on 3.5.0" - I'm afraid that doesn't mean much.

Show the exact commands you typed and the exact error message(s) you got. Also say what version you are upgrading *from* and *to*, and whether you originally installed Netbox using a release tarball or using a git checkout.

**Comment by mm3906078** [comment]
Thanks, i fixed the issue.

**Reply by candlerb** [reply]
Care to share what the problem was, in case someone else has the same?

**Reply by mm3906078** [reply]
I was just stupid and forget to pull the new version from git
